### DISCUSSION TITLE
Add MailNickName as a parameter to Set-PnPMicrosoft365Group ?

### DISCUSSION THREAD

**Discussion by kasperbolarsen** [pnp/powershell#3368] [Ideas] [answered]
Is there a reason ( besides nobody asking for it before? ;-) ) why I can't change the MailNickName using Set-PnPMicrosoft365Group?

**Comment by tberta** [comment, marked as answer]
You have been heard, updating MailNickname using Set-PnPMicrosoft365Group has been implemented in [v2.3.0](https://github.com/pnp/powershell/releases/tag/v2.3.0)

**Reply by kasperbolarsen** [reply]
Just as bonus info: I have asked the Graph team to add primaryEmail as a settable property.  We can change this property using Exchange powershell,  so it should available in Graph too.
